src/controllers/bookmarkController.js

```diff
@@ -0,0 +1,49 @@
+import BookmarkModel from '../helpers/bookmarkModel';
+/**
+ * @description class representing Bookmark Controller
+ * @class BookmarkController
+ */
+class BookmarkController {
+  /**
+   * @description - This method is responsible for bookmarking an article
+   * @static
+   * @param {object} request - Request sent to the router
+   * @param {object} response - Response sent from the controller
+   * @returns {object} - object representing response message
+   * @memberof BookmarkController
+   */
+  static async bookmark(request, response) {
+    const { slug } = request.params;
+    const userId = request.userData.payload.id;
+    await BookmarkModel.addBookmark(response, slug, userId);
+  }
+
+  /**
+   * @description - This method is responsible for getting a user's bookmarks
+   * @static
+   * @param {object} request - Request sent to the router
+   * @param {object} response - Response sent from the controller
+   * @returns {object} - object representing response message
+   * @memberof BookmarkController
+   */
+  static async getAllBookmark(request, response) {
+    const userId = request.userData.payload.id;
+    await BookmarkModel.getBookmarks(response, userId);
+  }
+
+  /**
+   * @description - This method is responsible for getting a user's bookmarks
+   * @static
+   * @param {object} request - Request sent to the router
+   * @param {object} response - Response sent from the controller
+   * @returns {void} - void representing response message
```

Comment:
"void representing response message"?

Per JSDoc, `@returns {void}` is used when the method/function has no return statement (like this right does).

See this 👇 

![image](https://user-images.githubusercontent.com/15332525/50248968-09e26b00-03dd-11e9-96e8-9ac954269212.png)

Also, this link would be helpful for knowing what JSDoc comment is valid => https://eslint.org/docs/rules/valid-jsdoc

Reply:
Thanks for sharing this, it was really helpful. I had been doing it all wrong. I have made corrections.